src/patternfly/components/Chip/chip.scss

```diff
@@ -24,6 +24,11 @@
   --pf-c-chip--m-overflow--c-button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
   --pf-c-chip--m-overflow--c-button--focus--BorderColor: var(--pf-global--secondary-color--100);
 
+  // read-only chip
+  --pf-c-chip--m-read-only--PaddingTop: var(--pf-global--spacer--form-element);
```

Comment:
It seems weird to me that the modified version is the simplest chip we have with all padding present..would it be wrong to have read only be the simplified chip component and then have a modifier for the chip with the "remove" button. I guess though the chip is always meant to come with the remove button so this should be fine... im just also thinking if we have chips with icons etc.... you can really put anything into a chip.

Reply:
>  I guess though the chip is always meant to come with the remove button so this should be fine

That's what I'm thinking, too. Plus that would be a breaking change. Can you give an example of where you would want to use an icon in a chip and how that would affect the layout?

Reply:
Ok lets stick with this then and avoid a breaking change. I have only seen icons used at the beginning and last element of the chip, but we dont have a case for Patternfly yet so all good